CHANGELOG.md

```diff
@@ -1,5 +1,229 @@
+    [pull/260]: https://github.com/cloudflare/wrangler/pull/260
+
+  - **Correctly pass Wasm module - [xtuc], [pull/261]**
+
+    To ensure that a wasm module is successfully passed between `wranglerjs` and `wrangler`, the wasm module is now encoded and decoded from base64, avoiding any potential loss of data.
+
+    [xtuc]: https://github.com/xtuc
+    [pull/261]: https://github.com/cloudflare/wrangler/pull/261
+
+  - **Check for `account_id` and `zone_id` before publishing - [xtuc], [issue/170] [pull/192]**
+
+    The `publish` subcommand in `wrangler` now ensures that you have an `account_id` and `zone_id` configured in your `wrangler.toml` file before attempting to publish, instead of failing during the publishing process.
+
+    [xtuc]: https://github.com/xtuc
+    [issue/170]: https://github.com/cloudflare/wrangler/issues/170
+    [pull/192]: https://github.com/cloudflare/wrangler/pull/192
+
+  - **Pass wranglerjs output as ref - [xtuc], [pull/227]**
+
+    When `wranglerjs` built a project, it incorrectly referred to the output of that build process without using a Rust reference - this PR fixes that issue and allows `wranglerjs` to correctly take your bundle, and your project's metadata, and put it all together in a nice little package to send up to the Cloudflare API. Hooray, working projects!
+
+    [xtuc]: https://github.com/xtuc
+    [pull/227]: https://github.com/cloudflare/wrangler/pull/227
+
+  - **Fix installer - [xtuc], [pull/195]**
+
+    Previously, `wranglerjs` would refer to a directory that potentially didn't exist on your machine, depending on your operating system. This change defaults the `wranglerjs` config file to `.wrangler` in your home directory, ensuring consistency throughout operating systems!
+
+    [xtuc]: https://github.com/xtuc
+    [pull/195]: https://github.com/cloudflare/wrangler/pull/195
+
+
+- ### 📖 Documentation
+
+  - **Clarified intro link in README - [tomByrer], [pull/257]**
+
+    [tomByrer]: https://github.com/tomByrer
+    [pull/257]: https://github.com/cloudflare/wrangler/pull/257
+
+  - **Make it more clear that you can install Wrangler though npm - [zackbloom], [pull/241]**
+
+    [zackbloom]: https://github.com/zackbloom
+    [pull/241]: https://github.com/cloudflare/wrangler/pull/241
+
+  - **Document (lightly) the Wrangler 1.0.0 release - [signalnerve], [pull/204]**
+
+    [signalnerve]: https://github.com/signalnerve
+    [pull/204]: https://github.com/cloudflare/wrangler/pull/204
+
+  - **Pkg readme - [ashleygwilliams], [pull/196]**
+
+    [ashleygwilliams]: https://github.com/ashleygwilliams
+    [pull/196]: https://github.com/cloudflare/wrangler/pull/196
+
+- ### 🔧 Maintenance
+
+  - **Remove pre-push hooks - [EverlastingBugstopper], [pull/308] **
+
+    Previous versions of Wrangler included pre-push hooks to ensure that code was linted before being pushed up to Git. This hook made it difficult to manage in-progress work, so the hooks have been removed.
```

Comment:
For some reason this is showing up bold, not sure if it’s an issue or not..

Reply:
oops, fixed